Assets/css/style.css

```diff
@@ -1,67 +1,71 @@
-
 .navbar {
     text-align: center;
 
 }
 
-.navbar-brand{
+.navbar-brand {
     font-size: 40px;
 }
 
-.nav-link{
+.nav-link {
     margin: 5px;
     border-radius: 5px;
     font-size: 20px;
 }
 
-@media(max-width:768px){
+@media(max-width:768px) {
 
     .nav-link {
 
         font-size: 20px;
 
     }
-    .navbar-brand{
+
+    .navbar-brand {
         font-size: 30px;
     }
 }
 
-.scl-icon{
+.scl-icon {
     width: 50px;
 }
 
-@media(max-width:474px){
-    .navbar-brand{
+@media(max-width:474px) {
+    .navbar-brand {
         font-size: 20px;
     }
+
     .nav-link {
 
         font-size: 15px;
 
     }
-    .scl-icon{
+
+    .scl-icon {
         width: 30px;
     }
 }
 
-.nav-link:hover{
+.nav-link:hover {
     color: #fff !important;
     background-color: #a5b1c2;
 }
-#main{
-    background-attachment:fixed;
+
+#main {
+    background-attachment: fixed;
     width: 100%;
     height: 90%;
-    background-image: linear-gradient(rgba(71, 177, 204, 0.644), rgba(110, 92, 212, 0.644)), url('/assets/img/main.jpg');
+    background-image: linear-gradient(rgba(71, 177, 204, 0.644), rgba(110, 92, 212, 0.644)), url('Assets/img/scl.gif');
```

Comment:
Change it as url("../img/scl.gif")
So this will work because the file path should be given from the CSS file .
Go back and go to img folder and go to gif file